pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/phimanifest/PhiManifestService.java

```diff
@@ -144,21 +145,43 @@ public PhiManifest generateDataForReport(@NonNull ElasticSearchParticipantDto pa
         phiManifest.setOrderStatus(mercuryOrderDto.getOrderStatus());
         phiManifest.setOrderStatusDate(DateTimeUtil.getDateFromEpoch(mercuryOrderDto.getStatusDate()));
 
-        String pediatricResponse = convertActivityAnswerValue((Boolean) participant.getParticipantAnswerInSurvey(
-                CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_STABLE_ID, SOMATIC_CONSENT_TUMOR_PEDIATRIC_QUESTION).orElse(false));
+        String pediatricResponse = convertBooleanActivityAnswerToReportValue(participant.getParticipantAnswerInSurvey(
+                CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_CODE, SOMATIC_CONSENT_TUMOR_PEDIATRIC_QUESTION_STABLE_ID));
         phiManifest.setSomaticConsentTumorPediatricResponse(pediatricResponse);
-        String assentAddendumResponse = convertActivityAnswerValue((Boolean) participant.getParticipantAnswerInSurvey(
-                CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_STABLE_ID, SOMATIC_ASSENT_ADDENDUM_QUESTION).orElse(false));
+
+        String assentAddendumResponse = convertBooleanActivityAnswerToReportValue(participant.getParticipantAnswerInSurvey(
+                CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_CODE, SOMATIC_ASSENT_ADDENDUM_QUESTION_STABLE_ID));
         phiManifest.setSomaticAssentAddendumResponse(assentAddendumResponse);
-        String consentAnswer = convertActivityAnswerValue(hasAdultParticipantConsentedToTumor(participant, ddpInstanceDto.getStudyGuid()));
-        phiManifest.setSomaticConsentTumorResponse(consentAnswer);
+        Optional<Boolean> consentAnswer = getAdultParticipantConsentedToTumorAnswer(participant, ddpInstanceDto.getStudyGuid());
+        if (consentAnswer.isPresent()) {
+            phiManifest.setSomaticConsentTumorResponse(convertBooleanToReportValue(consentAnswer.get()));
+        } else {
+            phiManifest.setSomaticConsentTumorResponse("");
+        }
         return phiManifest;
     }
 
-    private String convertActivityAnswerValue(boolean b) {
+    public static String convertBooleanToReportValue(boolean b) {
         return b ? "Yes" : "No";
     }
 
+    /**
+     * Given an answer to a boolean question, returns Yes, No, or a blank string
+     * to include in the PHI report.
+     */
+    public static String convertBooleanActivityAnswerToReportValue(Optional<Object> b) {
+        if (b.isPresent() && b.get() instanceof Boolean) {
```

Comment:
Not sure I understand, so maybe a comment or expanded Java doc for this method...
In addition to 'yes', 'no', and 'blank', the answer may have already been converted to a Boolean?

Reply:
It _might_ be a boolean.  Will see if I can improve this situation with generics.